src/adapters/aol.js

```diff
@@ -69,7 +56,7 @@ var AolAdapter = function AolAdapter() {
    * @param {ADTECHContext} context the context passed from aol
    */
   function _addBid(response, context) {
-    var bid = bidsMap[context.placement];
+    var bid = bidmanager.getPlacementIdByCBIdentifer(context.alias);
```

Comment:
I think you can map the bid within the module. I don't see a need to use the `bidmanager` method, it might be a leftover piece from a refactor a long time ago.